applepay/utils.py

```diff
@@ -1,20 +1,38 @@
+"""
+The utils in this module are used to verify and parse a decoded
```

Comment:
Ok, I will update this to describe the token, and not what has been done to it to get there.